src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
```

Comment:
Think about which variables should be local in the method and which should be class-level. 
Each time you call this method, a new random instance will be created. Let's make it a class variable and initialize it in the constructor